model/schema/core.yaml

```diff
@@ -144,6 +144,22 @@ classes:
       - generated_by
       - manufactured_by
 
+  Note:
+    description: >-
+      Free-text describing some attribute of an entity.
```

Comment:
I would prefer a description like: "Note object for capturing free-text describing some attribute of an entity, coupled with a 'note type', private boolean, and an optional list of references." Basically, I want to make sure that we don't just define the Note object as only free-text.

Reply:
Yes, that makes sense. Done.